src/db/schema.ts

```
@@ -60,3 +66,75 @@ export const verification = sqliteTable("verification", {
+`;
+
+export const resourceTags = sqliteTable(
+  "resource_tags",
+  {
+    resourceId: integer("resource_id")
+      .notNull()
+      .references(() => resources.id, { onDelete: "cascade" }),
+    tagName: text("tag_name").notNull(), // Storing tag name for fast lookups
+    // ✅ Define composite primary key correctly
+  },
+  (t) => ({
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)